source/hardware/controllers/numark_scratch.rst

```diff
@@ -0,0 +1,117 @@
+.. _numark_scratch:
+
+Numark Scratch
+==============
+
+The Numark Scratch is a 24-bit 2 channel mixer designed for the scratch DJ.
+It's equipped with an InnoFader crossfader and 8 performance pads across 3 modes, and
+6 dedicated software FX selectors with dual toggle triggers. The Scratch is DVS ready under Mixxx
+
+-  `Manufacturer’s product page <https://www.numark.com/product/scratch>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/numark-scratch>`__
+-  `Owner’s Manual <https://www.numark.com/images/product_downloads/Scratch_-_User_Guide_-_v1.2.pdf>`__
+-  `Driver Download Page <https://www.numark.com/product/scratch>`__
+-  `Mapping Forum Thread <https://mixxx.discourse.group/t/numark-scratch-mapping/25186>`__
+
+.. versionadded:: 2.4.0
+
+Drivers
+-------
+
+You can download the latest Windows driver from the `manufacturer’s website <https://www.numark.com/product/scratch>`__.
+The Scratch is class complaint on Mac and Linux.
+
+Audio Setup
+-----------
+
+The mapping relies on the following channel assignments (for line/phono
+input support, applying effects to the TR-S output, etc.):
+
+===================== ================
+Output Channels       Assigned to
+===================== ================
+1-2                   Master
+3-4                   Headphones
+===================== ================
+
+===================== ================
+Input Channels        Assigned to
+===================== ================
+1-2 (CH 1 Line/Phono) Vinyl Control 1
+3-4 (CH 2 Line/Phono) Vinyl Control 2
+5-6 (Mix)             Record/Broadcast
+===================== ================
+
+The microphone, and external inputs are mixed
+together in input channels 5-6, so Mixxx can record and broadcast them.
+
+The knobs for :hwlabel:`MASTER`, :hwlabel:`BOOTH`, :hwlabel:`MIC LEVEL`, :hwlabel:`MIC TONE` and :hwlabel:`CUE GAIN` are controlling the hardware mixer of the built-in audio interface.
+Hence, turning the knobs will not change values in the Mixxx :term:`GUI` and you’ll need to set the Mixxx knobs to their default values when using the controller:
+
+- Set the master/booth/headphones/microphone levels to 100% (knob center position)
+
+.. note::
+   You should assign the :guilabel:`Vinyl Control` input channels even if you do not intend to use timecode vinyl.
+   These channel assignments are also needed to make the :hwlabel:`CH 1/2 PC/LINE/PHONO` switches work (i. e. passing through line or phono input to the left/right deck).
```

Comment:
actually it was just "also" that made this a bit unclear to me
```suggestion
    The :hwlabel:`PC / LINE/PHONO` switches work only if the :guilabel:`Vinyl Control` input channels are assigned to software inputs. So even if you do not intend to use timecode vinyl but want to pass through line or phono inputs directly to the left/right deck, you need assign these channels in the Sound Hardware preferences.
```
Did I get that right? I.e. you refer to the hardware passthrough, not to the PASSTHROUGH option in Mixxx?

Reply:
Yep your version sound clearer